Packs/CortexXDR/Integrations/XDR_iocs/XDR_iocs.py

```diff
@@ -400,6 +439,38 @@ def sync(client: Client):
     return_outputs("sync with XDR completed.")
 
 
+def sync_for_fetch(client: Client, batch_size: int = 200):
+    """
+    Sync command runs in batches of 4,000 with total of 40,000 indicators in each sync.
+    Syncs the data in xsoar to xdr.
+    """
+    demisto.info("executing sync")
+    request_data: List[Any] = []
+    try:
+        full_query = create_query_with_end_time(to_date=get_integration_context().get('time'))
+        request_data = list(map(demisto_ioc_to_xdr, get_iocs_generator(size=batch_size,
+                                                                       stop_iteration=True,
+                                                                       query=full_query)))
+        if request_data:
+            integration_context = get_integration_context()
```

Comment:
notice that you just got integration context in line 450, I think you can use it here.
get_integration_context() is a bit of an expensive action, you should not make the code very complex in order to save it, but in easy place like this, you can reuse.

also, It is better to call this variable "last_run" this way you know this is a copy of the integration context before changes from this run.

Reply:
moved the log to other place so i will not have to get_integration_context again
fixed the rest of this comment